resources/windows/windows.pkr.hcl

```diff
@@ -93,7 +98,7 @@ locals {
     user  = "Administrator"
     images = [
       {
-        filter = "*Windows_Server-2019-English-Core-Base*"
+        filter = "Windows_Server-2019-English-Core-Base*"
```

Comment:
Context: Remove the * at the start, to ensure we don't run a `TPM-Windows_Server...etc..etc` AMI that causes this error:

```
==> win-builder-base.amazon-ebs.metasploit-windows-builder-1.0.9: Error launching source instance: UnsupportedOperation: The t2.medium instance type does not support an AMI with TPM version v2.0. Specify an instance type that supports TPM version v2.0, and try again.
```